qa/TL0_cpu_only/test_pytorch.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages="nose numpy>=1.17 pillow torch numba scipy librosa==0.8.1 tensorflow-gpu"
+
+target_dir=./dali/test/python
+
+test_body() {
+  # CPU only test, remove CUDA from the search path just in case
+  export LD_LIBRARY_PATH=""
+  export PATH=${PATH/cuda/}
+  nosetests --verbose test_dali_cpu_only.py
```

Comment:
```suggestion
  nosetests --verbose -m '(?:^|[\b_\./-])[Tt]est.*pytorch*' test_dali_cpu_only.py
```
?

Reply:
Or should we use `--attr 'pytorch'` vs. `--attr '!pytorch'` ? Right now we are running all tests under test_pytorch.sh